octopusdeploy_framework/resource_project_expand.go

```diff
@@ -74,9 +75,9 @@ func expandProject(ctx context.Context, model projectResourceModel) *projects.Pr
 		var gitAnonymousSettingsList []gitAnonymousPersistenceSettingsModel
 		diags := model.GitAnonymousPersistenceSettings.ElementsAs(ctx, &gitAnonymousSettingsList, false)
 		if diags.HasError() {
-			fmt.Printf("Error converting Git anonymous persistence settings: %v\n", diags)
+			tflog.Error(ctx, fmt.Sprintf("Error converting Git anonymous persistence settings: %v\n", diags))
```

Comment:
I modified these so they wrote out correctly to the terraform logs. In the tests these were writing to the console.